CHANGES.rst

```diff
@@ -1,4 +1,8 @@
-1.0 (unreleased)
+1.1 (unreleased)
+================
+
+
+1.0 (2019-9-20)
```

Comment:
```suggestion
1.0 (2019-09-20)
```

(to keep it ISO 8601 compatible (as well as the same length as the heading formatting)